specification/monitor/resource-manager/Microsoft.Insights/preview/2017-05-01-preview/metricDefinitions_API.json

```diff
@@ -49,6 +46,9 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
```

Comment:
I don't think the 2017-05-01-preview version of the API respects the metricNamespace parameter